src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.suppliers.Color;
+
+public abstract class Figure implements AreaCalculator, Printable {
+    private Color color;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    String getFormatted(double d) {
+        return String.format("%.2f", d);
+    }
```

Comment:
I think we can use `String.format` in each `draw` method in subclasses

Reply:
@hronograf  So, even if we have a similar implementation in every class, is it better to copy the method?